update libs

AndreyNeklesa opened the pull request.

AndreyNeklesa committed:
update libs

lexxorlov commented on libs/orm-core/src/extension/related-reference.ts:
```diff
@@ -12,8 +12,9 @@ export class RelatedReference<T extends RelatedEntity> {
+            //@TODO remove any in definition
             this.entity =
-                ((await wrap(this.entity, true).__em.findOne(this.entity.constructor.name, this.entity.id, {
+                ((await wrap(this.entity, true).__em.findOne(this.entity.constructor.name, this.id as any, {
```
I am not sure, replace this.entity.id to this.id is right way

bzhn-zfort committed:
fix: use raw queries after mikro ORM update

There were 500 error on login tries before.
More info:
https://mikro-orm.io/docs/upgrading-v5-to-v6

bzhn-zfort commented:
Merged to the branch I'm currently working on.
